jupyter-extension/setup.cfg

```diff
@@ -34,4 +34,5 @@ dev=
   pytest
   pytest-asyncio
   pytest-tornasync
+  pytest-jupyter
```

Comment:
Why was this added? I don't see it used in your changes

Reply:
@msteffen  added that one in his review, and I copied it.   I can try dropping it.

Reply:
Oh, yeah. When I tried creating a testing branch from the original pr, CI failed with an error about that as a missing dependency, so I added it 🤷. We should remove it if it’s not needed.